packages/studio-web/src/app/upload/upload.component.ts

```diff
@@ -332,6 +334,10 @@ export class UploadComponent implements OnInit {
         { timeOut: 10000 }
       );
     } else if (type === "text") {
+      if ((file.name.split('.').pop() === 'txt' && file.size > this.maxTxtSize) || (file.name.split('.').pop() === 'readalong' && file.size > this.maxRasSize)) {
```

Comment:
I don't think we restrict extensions to just .txt and .readalong.
I would rather do something like this:
```suggestion
      if ((file.name.split('.').pop() !== 'readalong' && file.size > this.maxTxtSize) || (file.size > this.maxRasSize)) {
```
so we don't leave any potential door open.

Reply:
you're right, we currently do accept .xml, but we shouldn't! .txt,.xml,.readalong

Reply:
That's not what I mean. I mean `foo.bar` should be accepted and assumed to be a plain text file. Unless we explicitly bar it, but I would find that exceedingly annoying.